Pipfile

```
@@ -6,7 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-aiidalab = "~=20.11.0b0"
+aiidalab = "~=21.04"
```

```suggestion
aiidalab = "~=21.04.0"
```
would probably be better

---

This makes me think that we should indeed switch to the semantic versioning, as the `aiidalab` package is not a meta-package anymore.

---

I don't think we need semantic versioning, but we could use a mixed approach where at least the point-releases should be compatible. But in principle we should not use the comaptible-release operator here, since there is no assumption about compatibility between releases. So this would actually be more appropriate:
```suggestion
aiidalab = "==21.04.01"
```

---

it is `21.04.1`, but I agree with your comment.